cg_hermes/config/balsamic.py

```diff
@@ -400,24 +400,24 @@
         "used_by": ["scout"],
     },
     frozenset({"vcf-filtered", "snv", "clinical-vcf-filtered"}): {
-        "tags": ["vcf-snv-clinical", "filtered"],
+        "tags": ["vcf", "vardict", "filtered"],
```

Comment:
Can we remove `vardict` here? Right now `clinical-vcf-filtered` is VarDict, but it won't be in future.

Reply:
```suggestion
        "tags": ["vcf", "filtered", "vcf-snv-clinical"],
```

Reply:
Think about this as "research" level VCF.

Reply:
Again we can only use that tag once. If you want this to be uploaded as a research file to scout you should use the tag `vcf-snv-research`